test-projects/gts/src/test.ts

```diff
@@ -1,4 +1,4 @@
-import { hi } from './await.gts';
```

Comment:
await.ts/js doesn't exist.

Reply:
Mmm,
So we need to also replace .ts extension in existsSync (and other places)

Reply:
does existsSync assume a .ts extension?

Reply:
I think typescript will check a few extensions if non is specified. ts, mts, tsx

Reply:
maybe it's not that bad of a perf impact to try multiple? idk